audb/core/publish.py

```diff
@@ -274,9 +274,6 @@ def publish(
     with :func:`audb.load_to` to ``db_root``.
     Afterwards you make your changes to that folder
     and run :func:`audb.publish`.
-    :func:`audb.publish` will then check
-    that the version of the files inside that folder
-    match the version given by ``previous_version``.
```

Comment:
This sentence I did not understood. But I also don't think we need it ;)

Reply:
I think it's only relevant for the rare case that you want to publish a new version of a database that is not based on the previous but an older version.

Reply:
OK, this should be covered by 

![image](https://user-images.githubusercontent.com/173624/147940863-4d6a477d-a361-4e43-bbbd-8ee08c41df7b.png)

So I think we can stay with the deleted sentence.